dashboard/src/js/components/Profile.js

```diff
@@ -45,6 +45,7 @@ function TabPanel(props) {
 }
 
 export default function ProfileDialog(props) {
+
```

Comment:
extremely minor - but i don't think you intended this blank line addition?

Reply:
thank you, paul, for catching that, I removed the extra blank line.